dotnet/samples/PlayFabExamples/Example02_Generative/SegmentSkill.cs

```diff
@@ -17,6 +17,99 @@ namespace PlayFabExamples.Example02_Generative;
 /// </summary>
 public sealed class SegmentSkill
```

Comment:
>SegmentSkill [](http://example.com/codeflow?start=20&length=12)

This is called "Skill" but it has nothing that reminds a skill...

See Example1 as for how skill should be written and invoked via the action planner